assets/i18n/en.json

```diff
@@ -1,4 +1,41 @@
 {
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE": "Create an Exploration",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT": "about a topic you care about.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK": "Earn feedback",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_FEEDBACK_TEXT": "to improve your exploration.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_HEADING": "About Oppia",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_1": "Oppia's mission is to help anyone learn anything they want in an effective and enjoyable way.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_2": "By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education &mdash; regardless of where they are or what traditional resources they have access to.",
+    "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_3": "So far, educators have created more than 8,000 of these lessons, which we call <em>explorations</em>. And they serve almost 250,000 students worldwide.",
```

Comment:
Done.
(This was an interesting challenge, but since angular-translate provides capabilities for variable replacement, I've used that to pass  the values. This way we can change it without affecting the rest of the text.)